Examples/WriteApiAsyncExample.cs

```diff
@@ -19,10 +19,10 @@ private class Temperature
             [Column(IsTimestamp = true)] public DateTime Time { get; set; }
         }
 
-        public static async Task Main(string[] args)
+        public static async Task Main()
         {
-            var influxDbClient = InfluxDBClientFactory.Create("http://localhost:9999",
-                "my-user", "my-password".ToCharArray());
+            using var influxDbClient = new InfluxDBClient("http://localhost:9999",
```

Comment:
Please rename var to `client`:
```suggestion
            using var client = new InfluxDBClient("http://localhost:9999",
```